src/views/DatasetManager/components/CreateDatasetButton/components/CreateDatasetWizard/CreateDatasetWizard.js

```diff
@@ -73,8 +73,9 @@ export const CreateDatasetWizard = ({ open, closeDialog }) => {
 
   const createDatasetAndCloseDialog = useCallback(() => {
     const values = methods.getValues();
-    if (datasetLocation === 'fromScratch') values.sourceType = 'None';
     DatasetsUtils.removeUndefinedValuesBeforeCreatingDataset(values);
+    if (datasetLocation === 'fromScratch') values.sourceType = 'None';
+    if (values.sourceType === DATASET_SOURCE_TYPE.LOCAL_FILE) values.source = null;
```

Comment:
Same minor suggestion, the second condition can never be true when the first one is true, so we can replace the second `if` by an `else if` to make it explicit